R/ari_stitch.R

```diff
@@ -92,6 +92,8 @@ ari_stitch <- function(images, audio,
   stopifnot(length(images) > 0)
   images <- normalizePath(images)
   output_dir <- normalizePath(dirname(output))
+  # Make a hard path
+  output <- file.path(output_dir, basename(output))
```

Comment:
Moved this up to be closer to `output_dir` assignment so the code flows from assigning `output_dir` to assigning `output_dir`